### ISSUE TITLE
CANNOT FIND the source code of “gen_candidate_sampling_ops”

### ISSUE THREAD

**Issue by JulieYao0754** [tensorflow/tensorflow#47584]
I’m reading the source code(python version) of sampled softmax loss(tf.nn.sampled_softmax_loss) since I want to write my own “sampled_values”, and it leads me to the function “log_uniform_candidate_sampler” in candidate_sampling_ops.py.

———————————————————————————————————-
@tf_export(
    'random.log_uniform_candidate_sampler',
    v1=[
        'random.log_uniform_candidate_sampler',
        'nn.log_uniform_candidate_sampler'
    ])
@dispatch.add_dispatch_support
@deprecation.deprecated_endpoints('nn.log_uniform_candidate_sampler')
def log_uniform_candidate_sampler(true_classes, num_true, num_sampled, unique,
                                  range_max, seed=None, name=None):

  seed1, seed2 = random_seed.get_seed(seed)
  return gen_candidate_sampling_ops.log_uniform_candidate_sampler(
      true_classes, num_true, num_sampled, unique, range_max, seed=seed1,
      seed2=seed2, name=name)
———————————————————————————————————-

HOWEVER, I cannot find the gen_candidate_sampling_ops.py in source code, which confused me.

[Link:](url)
https://github.com/tensorflow/tensorflow/blob/85c8b2a817f95a3e979ecd1ed95bff1dc1335cff/tensorflow/python/ops/candidate_sampling_ops.py#L99

**Comment by Saduf2019** [comment]
@JulieYao0754 
We see that the issue template has not been filled, could you please do so as it helps us analyse the issue [tf version, steps followed before you ran into this error or stand alone code to reproduce the issue faced]

**Comment by ymodak** [comment]
These (gen_*_.py) are automatically generated files and can be found in your local TF installation.
For me on PyCharm IDE it is located at:
```python
/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/tensorflow/python/ops/gen_candidate_sampling_ops.py
```

**Comment by JulieYao0754** [comment]
@ymodak thanks a lot!

**Comment by ymodak** [comment]
Are you satisfied with the resolution of your issue?
[Yes](https://goo.gl/forms/Oe0tEvODFRoI2gJF3)
[No](https://goo.gl/forms/fUjzOfrtkFbrOT8d2)
